backend/src/routes/user.ts

```diff
@@ -208,3 +210,26 @@ userRouter.post("/updateDetail", async (c) => {
     return c.json({ error: "Something went wrong" });
   }
 });
+
+userRouter.get("/memberSince/:id",async(c)=>{
+  const userId = await c.req.param("id");
+  const prisma = getDBInstance(c);
+  const joiningDate = await prisma.user.findFirst({
+    where:{
+      id:userId
+    },
+    select:{
+      creationDate:true
+    }
+  });
+  if (joiningDate?.creationDate) {
+    const formattedDuration = CalculateMemberSince(joiningDate.creationDate)
+    return c.json({
+      since:formattedDuration
+    })
+  }else{
+    return c.json({
+      error:"Cannot find Date"
+    })
+  }
+})
```

Comment:
This API isn't required. 
The Profile page makes an API call `api/v1/user/:userid` which already returns creationDate. It can be used as Member since.